doc/releases.md

```diff
@@ -186,6 +186,14 @@ informed perspective, such as a member of the NAN team.
 see a need to bump `NODE_MODULE_VERSION` then you should consult the TSC.
 Commits may need to be reverted or a major version bump may need to happen.
 
+*Note*: The Node.js ecosystem is reliant on ABI compatibility within a Semver
+Major release. To maintain ABI compatibility it is expected that production
```

Comment:
Nit: `Semver Major` -> `semver major` for consistency.

I dislike the term `semver major` because I think it's jargon that we understand but that not everyone should be expected to. However, I don't have a better alternative. Suggestions welcome, although it's certainly outside the scope of this PR. It's been bugging me for a long time.

Reply:
I think just `within a major version` is fine?